### PR TITLE
Migrate Gogs pulls as issue

### PR DESCRIPTION
**Pull request by 6543**
extent #14342 ...

Migrate pulls as pulls is not posible, because gogs do not provide needed information over api

### CODE REVIEW

File: modules/migrations/gogs.go [hunk 1]
```diff
@@ -220,12 +229,45 @@ func (g *GogsDownloader) getIssues(page int, state string) ([]*base.Issue, bool,
...
+	if err != nil {
+		if err.Error() == "404 Not Found" {
+			return nil, true, nil
+		}
+		return nil, false, err
+	}
+
+	issue := convertGogsIssue(gogsIssue)
+
+	issue.Title = "[Pull] " + issue.Title
```
Comment (zeripath) [inline modules/migrations/gogs.go:257]: I think we can actually just create PRs with missing heads or even just make fake ones ourselves

Comment (6543) [reply modules/migrations/gogs.go:257]: we have no base and no head

Comment (6543) [reply modules/migrations/gogs.go:257]: :point_up: with this all you get is a broken pull

Comment (6543) [reply modules/migrations/gogs.go:257]: @Cat7373 you as one who potentially use this migrator ... what are your thoughts about it?

Comment (6543) [reply modules/migrations/gogs.go:257]: @zeripath tested to create pull with only issue information: get `failed: Unsupported scheme: `

**Comment by lunny** [comment]
Since it's impossible, just disable the checkbox. I don't think there is any meaning to place an issue as pull request .

**Comment by 6543** [comment]
For documentation/archivation porpuse ...

**Comment by lunny** [comment]
> For documentation/archivation porpuse ...

No codes, commits and code comments there. They will only make users confusing.

File: modules/migrations/gogs.go (hunk 1 above)
Comment (zeripath) [reply modules/migrations/gogs.go:257]: yeah I forgot we don't have a base. We have a head it's simply refs/pulls/xxx/head - the base is the issue.

**Pull request closed without merging**